.github/workflows/buildAndTest.yml

```diff
@@ -142,15 +142,19 @@ jobs:
       - name: Install Clang & Clang++ and linker
         run: apt-get install -y clang lld
 
-      - name: Get Submodule Hash
+      - name: Get related-LLVM Submodule Hash
         id: get-submodule-hash
-        run: echo "hash=$(md5sum $(echo utils/clone-llvm.sh))" >> $GITHUB_OUTPUT
+        # Instead of using the git hash of LLVM, just use the good-enough hash
```

Comment:
If this is now a requirement on the clone-llvm.sh script, perhaps it should be documented?

Reply:
It it not "now a requirement", it has always been there. :-|
It took me some time to understand why this hash was working. This is why I commented (= documented :-) ) the CI here. :-)
I have just removed some tautological code.
But I agree that if some day we remove this textual git hash from `utils/clone-llvm.sh` it would break things, so I should also comment this script.
Thinking again about it and the discussion we had at lunch about reducing the technical debt, I think I will just remove this hack and look at the hash just in the `llvm` worktree after the step running `utils/clone-llvm.sh`.